.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Google JS style guide recommends 2 spaces without tabs and breaks at 80 cols maximum with indent at 2 spaces too. This would be correct :)